etna/home/models.py

```diff
@@ -1,7 +1,18 @@
 from django.db import models
 
 from wagtail.core.models import Page
+from wagtail.snippets.edit_handlers import SnippetChooserPanel
 
 
 class HomePage(Page):
-    pass
+    alert = models.ForeignKey(
```

Comment:
I can see the need to add the alerts to a lot of pages. Have you considered using a mixin?

Reply:
A mixin has been created for the alert field and added to the page models.